### ISSUE TITLE
Allow user to delete account

### ISSUE THREAD

**Issue by TimStallard** [TimStallard/NextGenPlanner#9]
Confirm via email with deletion link/code?

**Comment by isaacrg** [comment]
@TimStallard Perhaps this could be done through generating a new api key for that email. I recommend emailjs+mandrill for that

**Comment by TimStallard** [comment]
Yeah, I still need to get round to moving everything over to Mongoose, then I'll add email confirmation & deletion :)

**Comment by isaacrg** [comment]
@TimStallard Cool :)
